packages/gatsby/src/schema/types/sort.ts

```diff
@@ -100,10 +52,10 @@ const convert = (
       }
     }
 
-    const type = getNamedType(fieldConfig.type)
+    const type = fieldConfig.type
     if (type instanceof GraphQLInputObjectType) {
       if (depth < MAX_SORT_DEPTH) {
-        const typeComposer = schemaComposer.getAnyTC(
+        const typeComposer = schemaComposer.getIFTC(
```

Comment:
should this be getITC and not getITFC?

getITC is for inputs and getITFC is for interfaces

I didn't look up code around it much yet, just the type instanceof GraphQLInputObjectType part

Reply:
`typeComposer` in this function is invoke `getFieldExtension` getter method like below line.
`typeComposer.getFieldExtension(fieldName, "sortable")`
and this customized extension field is defined in `nodeInterfaceTC: IFTC` at `./node-interface.js` like below code.

```
extensions: {
          searchable: SEARCHABLE_ENUM.SEARCHABLE,
          sortable: SORTABLE_ENUM.SORTABLE,
          needsResolve: true,
        }
```

I would appreciate any better way🙏🏻

Reply:
I am out of my depth here. I will need to leave proper review for our GraphQL folks